docs/admin/templates/managing-templates/devcontainers/add-devcontainer.md

```diff
@@ -0,0 +1,143 @@
+# Add a dev container template to Coder
+
+A Coder administrator adds a dev container-compatible template to Coder
+(Envbuilder). This allows the template to prompt for the developer for their dev
+container repository's URL as a
+[parameter](../../extending-templates/parameters.md) when they create their
+workspace. Envbuilder clones the repo and builds a container from the
+`devcontainer.json` specified in the repo.
+
+You can create template files through the Coder dashboard, CLI, or you can
+choose a template from the
+[Coder registry](https://registry.coder.com/templates?tag=devcontainer):
+
+<div class="tabs">
+
+## Dashboard
+
+1. In the Coder dashboard, select **Templates** then **Create Template**.
+1. To use a
+   [starter template](https://github.com/coder/coder/tree/main/examples/templates),
+   select **Choose a starter template**.
+
+   - Select **From scratch** to create a new template and enter information and
+     file contents manually.
```

Comment:
They can also directly import the devcontainer built-in templates without copying Patse.
<img width="1312" alt="image" src="https://github.com/user-attachments/assets/f51fe216-ec72-4549-a2f0-36e6b3f2f3cf">